File: angular/argon-design-system-angular-master/src/app/interface/charge.ts

import { Billing } from "./billing";


export interface Charge{

    source: string;
    amount: number;
    description: string;
    paymentMethod: string;
    currency: Date;
    redirect3dsUri: Date;
    cvv: string;
    saveCard: string;
    billing: Billing;

}

File: scoolTRX/src/main/java/com/flitx/scool/model/Charge.java
package com.flitx.scool.model;


import lombok.Data;
import org.springframework.data.mongodb.core.mapping.Document;

import java.math.BigDecimal;

@Data
@Document("Charge")
public class Charge {

    private String source;
    private BigDecimal amount;

    private String description;
    private String paymentMethod;

    private String currency;
    private String redirect3dsUri;

    private String cvv;
    private String saveCard;

    private Billing billing;



}
